packages/p2p/src/components/my-ads/my-ads.scss

```diff
@@ -700,3 +700,15 @@
         }
     }
 }
+
+.edit-ad__offer-amt {
+    .dc-input__label {
+        color: var(--text-general);
+        -webkit-text-fill-color: var(--text-general);
+    }
```

Comment:
can we use the color prop of `Text` instead?

Reply:
This CSS is to override the color of the label of a disabled field. Also The label prop of `Input` component takes string. Hence cannot use text